imdb.demo/src/test/java/com/springboot/imdb/demo/ApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.springboot.imdb.demo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
Write Test Cases and use @RunWith(Junit4.class) something like this instead @SpringBootTest